example/hands-on/config/bucky_config.yml

```diff
@@ -0,0 +1,6 @@
+:parallel_num: 4
```

Comment:
why do you commit this file?

Reply:
I commit all files which are created in this example.